packages/members-api/index.js

```diff
@@ -95,11 +95,21 @@ module.exports = function MembersApi({
         if (!email) {
             return null;
         }
+        /** Since we pass labels as comma-seperated string in token, it needs proper formatting for member creation*/
```

Comment:
What's the reason we don't pass an array?

Reply:
The primary reason I avoided passing an array was to avoid increasing the length of payload which in turn increases the length of token/magic-link. Will definitely need to change this though to allow string arrays as we want to allow `,` in the label name. 

We will still need the formatting of array to convert it to a label object array with `[{"name": "Something"]` structure here as we can't pass simple string array to member creation call. 

@allouis Should we cap off the number/length of labels that can be passed like this as a concern for token/magic-link size increase, or is that something which is not much of an issue ? I remember previous discussion where we wanted to keep the length of link in check.

Reply:
@allouis Cleaned up the array handling here and also moved the string to label object formatting in `user.create` method here - https://github.com/TryGhost/Members/pull/124/files#diff-3daeef67d07a2a0f94c89a86cafcede9R158-R167 . 

In the API layer, this serialization is handled by input serializer, and since we bypass that here and access model directly, we'll need to pass the `labels` in the correct format.

Reply:
> Should we cap off the number/length of labels that can be passed like this as a concern for token/magic-link size increase, or is that something which is not much of an issue ? I remember previous discussion where we wanted to keep the length of link in check.

This is something that needs to be discussed with Zimo/John I think :relaxed:

Reply:
Ah Yeah, I'll follow up in this, thanks :) Lmk if the changes looks sane here and I'll start locking labels stuff 😄